go.mod

```diff
@@ -21,23 +23,25 @@ require (
 	github.com/rancher/apiserver v0.0.0-20240503193545-2e1b0ddd9791
 	github.com/rancher/dynamiclistener v0.5.0-rc6
 	github.com/rancher/kubernetes-provider-detector v0.1.5
+	github.com/rancher/lasso v0.0.0-20240603075835-701e919d08b7
 	github.com/rancher/norman v0.0.0-20240503193601-9f5f6586bb5b
 	github.com/rancher/remotedialer v0.3.2
+	github.com/rancher/wrangler v1.1.2
```

Comment:
We've already imported wrangler/v2 below. In addition, wrangler at this version seems to support only up to k8s 0.25, which is much lower than what this PR targeted. I would recommend that this import be removed, and any functionality which relies on it be changed to use wrangler v2.